act/retrievals/planck.py

```diff
@@ -0,0 +1,49 @@
+import numpy as np
+
+
+def planck(wnum=None, radiance=None, temperature=None):
```

Comment:
do we want to rename this to something else more descriptive of what it's doing?  right now we're calling planck.planck()